app/src/main/java/com/jmstudios/redmoon/Config.kt

```diff
@@ -84,6 +84,10 @@ object Config : Preferences(appContext) {
         activeProfile.run { if (it != lowerBrightness) activateProfile(copy(lowerBrightness = it)) }
     }
 
+    var monochrome by BooleanPreference(R.string.pref_key_monochrome, false) {
+        activeProfile.run { if (it != monochrome) activateProfile(copy(monochrome = it)) }
```

Comment:
I'm sure this works, since it matches the other ones, but ugh, I was so hyperfixated on making code concise (specifically as measured by lines of code) when I wrote this.

Today I'd write it more like this (assuming I have correctly inferred `it`; I did this in GitHub webui):

```suggestion
        if (it != monochrome) {
          activateProfile(activeProfile.copy(monochrome = it))
        }
```

No need to change this, unless you want to change the others too, which you are more than welcome to.